src/universal/modules/welcome/components/WelcomeWizardForms/Step3InviteTeam.js

```diff
@@ -73,97 +102,81 @@ const Step3InviteTeam = props => {
       }
     } else if (data) {
       // Dispatch sequential success thunk:
-      dispatch(
-        Promise.all([
-          dispatch(show(emailInviteSuccess)), // trumpet our leader's brilliance!
-          dispatch(push(`/team/${teamId}`)),  // redirect leader to their new team
-          dispatch(destroy('welcomeWizard')), // Bye bye form data!
-        ])
-      );
+      dispatch((thunkDispatch) => Promise.all([
```

Comment:
Same as before. Route changes make multiple actions difficult. Think seems to be an adequate work-around. Very open to suggestions!